### ISSUE TITLE
Shouldn't be using the Timestamp() method when posting events to Loggr

### ISSUE THREAD

**Issue by davideweaver** [imobile3/Loggr.Extensions.Logging#4]
In the file LoggrLogger.cs, you create an event and set the timestamp before posting it. Your code should not be doing that and it will cause problems if the time on your server is not in-sync with the time on our servers. 

The TimeStamp() method is used when you are queuing events on your side (typically in an offline scenario) and sent at a time different than when it was logged. In the case of an application that was offline when an event occurred and it waits until it is back online to post the event to Loggr, then TimeStamp() can be used to indicate that. 

I hope that is helpful. We've had a customer run into this problem.

**Comment by kspearrin** [comment]
We implemented `Timestamp(DateTime.UtcNow)` because we were seeing inconsistencies with logs on our servers and the timestamps being shown in Loggr (few seconds delay).

We need to be able to have Loggr show exactly when the event occurred on our client, not when Loggr processed it. Is there some other way to do this?

**Comment by davideweaver** [comment]
That’s the way you would want to do that. Unless your server time is too out of sync with our servers it will work. There must be another issue with the customer I referred to. Thanks for getting back to me.

On Nov 9, 2016, at 2:02 PM, Kyle Spearrin <notifications@github.com<mailto:notifications@github.com>> wrote:

We implemented Timestmap(DateTime.UtcNow) because we were seeing inconsistencies with logs on our servers and the timestamps being shown in Loggr (few seconds delay).

We need to be able to have Loggr show exactly when the event occurred on our server, not when Loggr processed it. Is there some other way to do this?
